tests/v3_api/test_bkp_restore.py

```diff
@@ -0,0 +1,113 @@
+import base64
+
+import pytest
+from .common import *  # NOQA
+
+CLUSTER_NAME = os.environ.get("RANCHER_CLUSTER_NAME", "")
```

Comment:
This is already defined in common.py. Can be removed.

Reply:
removed this env variable